### ISSUE TITLE
Expose loading functions from main module interface

### ISSUE THREAD

**Issue by vigji** [neuroinformatics-unit/movement#361]
**Is your feature request related to a problem? Please describe.**
I find it a bit cumbersome to remember every time that loading functions need to be imported from `movement.io.load_poses` or whatever other module.

**Describe the solution you'd like**
I'm sure you people have been discussing about this and maybe do not want to prematurely expose functions from a module and then force people to refactor their code if you move those functions/rename etc. I guess this must be the issue, as otherwise since any `movement`-related project would at some point import functions from the `io` module additional dependencies loading time should not really be a factor here.

However, since every project indeed starts with such imports, I would really like to be able to use a function to load my files directly from the movement module interface (`movement.load_file`? `movement.load_pose`?)

What are your thoughts on this?

**Comment by adamltyson** [comment]
Sounds like a good idea to me.

**Comment by niksirbi** [comment]
Agree the current imports are cumbersome, especially for io.
Just to clarify, is this sth like the interface you envision?

```python
import movement as mvt

# instead of movement.io.load_poses.from_file()
mvt.load_poses(file_path)

# instead of movement.io.load_bboxes.from_file()
mvt.load_bboxes(file_path)
```

**Comment by vigji** [comment]
yup! I see how that would entail renaming functions from current load_poses module, but maybe that can be done gradually with duplication and warnings. I feel it would be important to lower friction for an entry user :)

**Comment by niksirbi** [comment]
I'm closing this issue because we now have the `load_dataset` function exposed from `movement.io`, since [v0.14.0](https://github.com/neuroinformatics-unit/movement/releases/tag/v0.14.0).

It's not quite in the main module interface, but close enough.

```python
from movement.io import load_dataset

ds = load_dataset("path/to/file.h5", source_software="DeepLabCut", fps=30)

# or equivalently

import movement as mvt

ds = mvt.io.load_dataset("path/to/file.h5", source_software="DeepLabCut", fps=30)
````

Since [v0.16.0](https://github.com/neuroinformatics-unit/movement/releases/tag/v0.16.0), even the `source_software` parameter is not strictly necessary.

I'm counting that as addressing the 'spirit', if not the 'letter' of this issue.
Feel free to re-open if you think there are still reasons for exposing it from the main module itself, i.e.:

````python
from movement import load_dataset
````
